crates/test-utils/src/messages.rs

```diff
@@ -175,6 +175,45 @@ pub async fn make_transactions_with_wallet_context(
     res
 }
 
+pub async fn make_transactions_with_wallet_context_and_rgp(
+    context: &mut WalletContext,
+    max_txn_num: usize,
+) -> Vec<VerifiedTransaction> {
+    let recipient = get_key_pair::<AuthorityKeyPair>().0;
+    let accounts_and_objs = get_account_and_gas_objects(context).await;
+    let mut res = Vec::with_capacity(max_txn_num);
+    let client = context.get_client().await.unwrap();
+    let gas_price = client
+        .governance_api()
+        .get_reference_gas_price()
+        .await
+        .unwrap();
+    for (address, objs) in &accounts_and_objs {
```

Comment:
afaik, in cluster test we won't use multiple accounts to generate txes, but only the active address

Reply:
We do use this in FullNodeExecuteTransactionTest to make txes though.